packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -162,7 +162,7 @@ export const FeedContainer = ({
             <span className="flex flex-row flex-1 mt-4">
               <SearchBarSuggestionList
                 {...suggestionsProps}
-                className="hidden tablet:flex overflow-hidden flex-1 mr-3"
+                className="hidden tablet:flex overflow-hidden flex-1 tablet:flex-none mr-3"
```

Comment:
You have `tablet:flex` and `tablet:flex-none`

Not sure which one get's rendered 👀
Maybe it's russian roulette

Reply:
The `flex-none` is not conflicting with the `flex` as the former only unsets the `flex-1` but keeps the `display: flex` intact.

Reply:
Ah right, :) tailwind naming brain fart

Reply:
Just like ellipsis classes. Not sure which one to use, `text-ellipsis` or `overflow-ellipsis` 😂